Tests.Integration.Presentation.Web/Organizations/V2/OrganizationGridConfigurationAPIV2Test.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Organizations.V2
+{
+    public class OrganizationGridConfigurationApiV2Test : WithAutoFixture
+    {
+
+        [Fact]
+        public async Task LocalAdminCanSaveGridConfig()
+        {
+            var (org, cookie) = await CreatePrerequisites();
+
+            var columns = CreateTestColumns().ToList();
+            var saveResponse = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, cookie);
+            Assert.Equal(HttpStatusCode.OK, saveResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task RegularUserCanGetGridConfiguration()
+        {
+            //Admins saves a column configuration
+            var columns = CreateTestColumns().ToArray();
+            var (org, localAdminCookie) = await CreatePrerequisites();
+            _ = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, localAdminCookie);
+
+            //User fetches column configuration
+            var (_, _, userCookie) = await HttpApi.CreateUserAndLogin(CreateEmail(), OrganizationRole.User, org.Id);
+            var userResponse =
+                await OrganizationGridConfigurationTestHelper.SendGetConfigurationRequestAsync(org.Uuid, userCookie);
+            var userResponseBody = await userResponse.ReadResponseBodyAsAsync<OrganizationGridConfigurationResponseDTO>();
+
+            Assert.Equal(HttpStatusCode.OK, userResponse.StatusCode);
+            foreach (var column in userResponseBody.VisibleColumns)
```

Comment:
the order of the columns should be the same as the order of the saved columns, right? If so check the order as well.

If it's ordered by a different property check that as well

Reply:
Same comments for the RegularUserSeesLocalAdminChanges + RegularUserSeesLocalAdminChanges does the same thing as this one.

This one then can be deleted, since the RegularUserSeesLocalAdminChanges consists of the same logic as this test

Reply:
I think they just depend on the Index, but i will check that the index matches aswell